db/db_test_util.cc

```diff
@@ -1211,7 +1212,8 @@ std::string DBTestBase::DumpSSTableList() {
 
 void DBTestBase::GetSstFiles(Env* env, std::string path,
                              std::vector<std::string>* files) {
-  env->GetChildren(path, files);
+  // Path may not exist
+  env->GetChildren(path, files).PermitUncheckedError();
```

Comment:
Would the Status ever be non-OK and the files be non-empty?  Why not check if ok() and then do the following code?